VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/judge-hearing-list/judge-hearing-list.component.ts

```diff
@@ -75,6 +84,19 @@ export class JudgeHearingListComponent implements OnInit, OnDestroy {
       });
   }
 
+  postJudgeEvent() {
+    const eventStatusDetails = this.eventStatusCache.get();
+    if (eventStatusDetails) {
+      this.videoWebService.raiseParticipantEvent(eventStatusDetails.ConferenceId,
+        new UpdateParticipantStatusEventRequest({
```

Comment:
Same concern as before, this is endpoint seems to function different to what you expect it to.

Reply:
Resolved as above by re-using this method with an updated reason.